Flux surface test fails occasionally in the CI and reliably locally

## Describe the bug

Locally on my home machine ```pytest tests/equilibria/test_flux_surfaces.py::test_johner``` fails. I've seen it fail on a number of occasions in the CI as well.

## Steps to reproduce

Steps to reproduce the behaviour:
```pytest tests/equilibria/test_flux_surfaces.py```

or run the CI enough times.

## Expected behaviour

The test should pass, and I'm not sure why it does occasionally and doesn't occasionally.

## Screenshots / Tracebacks

```python
self = <tests.equilibria.test_flux_surfaces.TestClosedFluxSurface object at 0x7f635b3d7850>

    def test_johner(self):
    
        R_0, z_0, a, kappa_u, kappa_l, delta_u, delta_l, a1, a2, a3, a4 = (
            7,
            0,
            2,
            1.9,
            1.6,
            0.4,
            0.33,
            -20,
            5,
            60,
            30,
        )
        fs = flux_surface_johner(
            7, 0, 2, kappa_u, kappa_l, delta_u, delta_l, a1, a2, a3, a4, n=1000
        )
        fs.close()
        fs = ClosedFluxSurface(fs)
        assert np.isclose(fs.major_radius, R_0)
        assert np.isclose(fs._z_centre, z_0)
        assert np.isclose(fs.minor_radius, a)
        assert np.isclose(fs.kappa, np.average([kappa_l, kappa_u]))
        assert np.isclose(fs.kappa_upper, kappa_u)
        assert np.isclose(fs.kappa_lower, kappa_l)
>       assert np.isclose(fs.delta, np.average([delta_l, delta_u]))
E       assert False
E        +  where False = <function isclose at 0x7f637042e9d0>(0.4210729575484019, 0.365)
E        +    where <function isclose at 0x7f637042e9d0> = np.isclose
E        +    and   0.4210729575484019 = <bluemira.equilibria.flux_surfaces.ClosedFluxSurface object at 0x7f635b32edb0>.delta
E        +    and   0.365 = <function average at 0x7f63854f8700>([0.33, 0.4])
E        +      where <function average at 0x7f63854f8700> = np.average

```

## Environment (please complete the following information)

- OS: 20.04 [develop] reliably, and CI occasionally

## Additional context

Add any other context about the problem here.

--- a/tests/equilibria/test_flux_surfaces.py
+++ b/tests/equilibria/test_flux_surfaces.py
@@ -127,16 +127,17 @@ class TestClosedFluxSurface:
             1.6,
             0.4,
             0.33,
-            -20,
-            5,
-            60,
+            0,
+            0,
+            45,
             30,
         )
         fs = flux_surface_johner(
-            7, 0, 2, kappa_u, kappa_l, delta_u, delta_l, a1, a2, a3, a4, n=1000
+            R_0, z_0, a, kappa_u, kappa_l, delta_u, delta_l, a1, a2, a3, a4, n=1000
         )
         fs.close()
         fs = ClosedFluxSurface(fs)
+
         assert np.isclose(fs.major_radius, R_0)
         assert np.isclose(fs._z_centre, z_0)
         assert np.isclose(fs.minor_radius, a)

